client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -185,6 +188,16 @@ class Dashboard extends Component {
 			return <RequiredPluginsInstallView site={ selectedSite } />;
 		}
 
+		if (
+			finishedInstallOfRequiredPlugins &&
+			isCalypsoStoreDeprecatedOrRemoved &&
+			shouldRedirectAfterInstall
+		) {
+			// Redirect to Core UI setup after finish installation.
+			this.redirectToWoocommerceSetup( selectedSite );
+			return <RequiredPluginsInstallView site={ selectedSite } />;
```

Comment:
Return `RequiredPluginsInstallView` to prevent from loading screen disappearing before redirect